app/auto_responder.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package app
+
+import (
+	l4g "github.com/alecthomas/log4go"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+func (a *App) SendAutoResponse(channel *model.Channel, receiver *model.User) {
```

Comment:
@esethna Should the auto response be sent as a reply to the original message? As it is, you won't see the response on mobile if you have the thread view open

Reply:
That's a good point. @miguelespinoza can we please change this so it responds as a reply to the DM message, rather than a parent post?